src/slave/containerizer/mesos/isolators/xfs/disk.cpp

```diff
@@ -136,6 +136,12 @@ static Option<Bytes> getSandboxDisk(
 
 Try<Isolator*> XfsDiskIsolatorProcess::create(const Flags& flags)
 {
+  Try<Nothing> enable = xfs::enableQuota(flags.work_dir);
```

Comment:
@bmahler Should there be a flag and/or environment variable to explicitly disable quota enforcement? Can't recall where we landed on this.